loopchain/baseservice/timer_service.py

```diff
@@ -129,6 +130,19 @@ def add_timer(self, key, timer):
         if timer.is_run_at_start:
             self.restart_timer(key)
 
+    def add_timer_convenient(self, timer_key, duration, is_repeat=False, callback=None, callback_kwargs=None):
```

Comment:
It seems a shortcut of `add_timer`. But it has another logic that 
```python
if timer_key not in self.__timer_list
```

I think the method should create new timer and add that timer without 'if statement'